src/karts/rescue_animation.cpp

```diff
@@ -37,6 +37,7 @@
 
 #include <algorithm>
 
+#include "graphics\camera_normal.hpp"
```

Comment:
Should be a forward-slash for better portability, only windows uses backslashes in paths